City-of-Pori/pori-kada
# PORI-315 Removed font sizing from table paragraphs

To test:

1. drush cc all
2. Go to edit any page content and add a table. Write something inside a cell and press enter and write something more (there should be paragraphs now inside cells)
3. Save the content and check that the font size is same with table cell content that is inside paragraph or not.
